docs/ref/urlresolvers.txt

```diff
@@ -132,7 +132,7 @@ For example:
 
 A lazily evaluated version of `reverse()`_.
 
-.. function:: reverse_lazy(viewname, urlconf=None, args=None, kwargs=None, current_app=None)
+.. function:: reverse_lazy(viewname, urlconf=None, args=None, kwargs=None, current_app=None, *, query=None, fragment=None)
```

Comment:
Can you add the same versionchanged note?
```
.. versionchanged:: 5.2
    The ``query`` and ``fragment`` arguments were added.
```

Reply:
Right, done!